data/1.0/_index.md

```diff
@@ -34,24 +34,41 @@ garage.save(ferrari);
 
 ### New features, enhancements or additions
 
-* The CrudRepository feature
-* Allows pagination on repository with PageableRepository
+* Define the Repository pattern
+* Allows pagination on Repository
+* Enhance type safety with a StaticMetamodel
+* Define platform integrations with CDI, Persistence, NoSQL, Transactions, and Validation
+* Produce a TCK with the ability to run against the following matrix:
+  * Profiles: none, core, web, full
+  * Entities: any, persistence, nosql
```

Comment:
I am not sure what `any` means. If you have `any`, why do you need `persistence` and `nosql`. I would state: `persistence`, `nosql`.